assets/js/aplus.js

```diff
@@ -463,9 +463,14 @@ $(function() {
             this.element.hide();
             var self = this;
             this.selection = this.widget.find(this.settings.selection_selector);
+            this.api_url = self.element.attr("data-users-api-url");
             this.selection_li = this.selection.find("li").remove();
             this.element.find("option:selected").each(function(index) {
-                self.addSelection($(this).attr("value"), $(this).text());
+              $.ajax({
+                url: self.api_url+$(this).attr("value"),
+              }).done( function( data ) {
```

Comment:
We haven't used extra whitespace around this kind of anonymous functions, have we? Have you seen this elsewhere in A+ code? Without whitespace, the single line would be `}).done(function(data) {` instead of `}).done( function( data ) {`